test/pt2_to_circle_test/test_model.py

```diff
@@ -31,15 +31,29 @@ def load_tests(loader, tests, pattern):
     https://docs.python.org/ko/3.13/library/unittest.html#load-tests-protocol
 
     Load only tests which matches the pattern given by the user, among them which are defined in `model` directory.
+    For exact model matching, use exact directory name instead of pattern matching.
     """
     suite = TestSuite()
-    matches = fnmatch.filter(
-        os.listdir(str(testdir) + "/modules/model"), loader.testNamePatterns[0]
-    )
+
+    test_model = os.environ.get("CCEX_TEST_MODEL")
+
+    if test_model:
+        matches = fnmatch.filter(
+            os.listdir(str(testdir) + "/modules/model"), test_model
+        )
+    elif hasattr(loader, "testNamePatterns") and loader.testNamePatterns:
+        # Fall back to pattern matching for backward compatibility
+        pattern_name = loader.testNamePatterns[0]
+        matches = fnmatch.filter(
+            os.listdir(str(testdir) + "/modules/model"), pattern_name
+        )
+    else:
+        # No filtering specified - return empty suite
+        return suite
```

Comment:
Are these lines really run?
```suggestion
    assert test_model
    matches = fnmatch.filter(
        os.listdir(str(testdir) + "/modules/model"), test_model
    )
```

Reply:
Okay, I'll update it.